lib/classes/CLI.js

```diff
@@ -52,24 +50,6 @@ class CLI {
     return true;
   }
 
-  asciiGreeting() {
-    let art = '';
-    art = `${art} _______                             __${os.EOL}`;
-    art = `${art}|   _   .-----.----.--.--.-----.----|  .-----.-----.-----.${os.EOL}`;
-    art = `${art}|   |___|  -__|   _|  |  |  -__|   _|  |  -__|__ --|__ --|${os.EOL}`;
-    art = `${art}|____   |_____|__|  \\___/|_____|__| |__|_____|_____|_____|${os.EOL}`;
-    art = `${art}|   |   |             The Serverless Application Framework${os.EOL}`;
-    art = `${art}|       |                           serverless.com, v${version}${os.EOL}`;
-    art = `${art} -------'`;
-
-    this.consoleLog(chalk.yellow(art));
-    this.consoleLog('');
-  }
-
-  printDot() {
-    process.stdout.write(chalk.yellow('.'));
-  }
-
   log(message, entity, opts) {
```

Comment:
We should keep both `log` and `consoleLog` methods here, right? Otherwise it would be a hard breaking change for plugins (currently there's a TODO to remove the whole class)

Reply:
Yes, we should keep it.

Technically this PR should be about removing all log constructs that rely on `legacy` interface from `@serverless/utils` (which should also be removed from there in context of v6. release of utils).

Those functions do not rely on it, therefore they're out of scope